src/main/java/net/dries007/tfc/objects/te/TELamp.java

```diff
@@ -0,0 +1,127 @@
+package net.dries007.tfc.objects.te;
+
+import javax.annotation.Nonnull;
+import javax.annotation.Nullable;
+
+import net.minecraft.block.state.IBlockState;
+import net.minecraft.inventory.InventoryHelper;
+import net.minecraft.item.ItemStack;
+import net.minecraft.nbt.NBTTagCompound;
+import net.minecraft.util.EnumFacing;
+import net.minecraft.util.math.BlockPos;
+import net.minecraft.world.World;
+import net.minecraftforge.common.capabilities.Capability;
+import net.minecraftforge.fluids.FluidStack;
+import net.minecraftforge.fluids.FluidTank;
+import net.minecraftforge.fluids.capability.CapabilityFluidHandler;
+import net.minecraftforge.fluids.capability.IFluidHandler;
+import net.minecraftforge.fluids.capability.IFluidHandlerItem;
+
+import net.dries007.tfc.ConfigTFC;
+import net.dries007.tfc.objects.fluids.capability.FluidHandlerSided;
+import net.dries007.tfc.objects.fluids.capability.FluidTankCallback;
+import net.dries007.tfc.objects.fluids.capability.IFluidHandlerSidedCallback;
+import net.dries007.tfc.objects.fluids.capability.IFluidTankCallback;
+import net.dries007.tfc.objects.items.itemblock.ItemBlockMetalLamp;
+
+public class TELamp extends TETickCounter implements IFluidTankCallback, IFluidHandlerSidedCallback
+{
+    public final static int CAPACITY = ConfigTFC.GENERAL.metalLampCapacity;
```

Comment:
This defeats the purpose of making it a config option. You need to query it from the config, not just statically reference it.